packages/inventory-vulnerabilities/src/Components/Remediation.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable camelcase */
+
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications';
+import RemediationButton from '@redhat-cloud-services/frontend-components-remediations/RemediationButton';
+import routerParams from '@redhat-cloud-services/frontend-components-utilities/files/RouterParams';
+import propTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Remediation extends Component {
+    remediationProvider = selectedCves => {
+        if (!selectedCves || selectedCves.size === 0) {
+            return false;
+        }
+
+        return {
+            issues: [ ...selectedCves ].map(cve => ({ id: `vulnerabilities:${cve}`, description: cve })),
+            systems: [ this.props.systemId ]
+        };
+    };
+
+    shouldComponentUpdate(nextProps) {
```

Comment:
Same as above, the component should re render on each prop and state update. You are preventing to do so in this function.